compaction/compaction.cc

```diff
@@ -752,6 +753,7 @@ class compaction {
                 continue;
             }
 
+            compaction_size += sst->data_size();
             // We also capture the sstable, so we keep it alive while the read isn't done
```

Comment:
maybe add it directly into _cdata.compaction_size (to avoid local variable)?

on exception during setup phase, we drop the compaction anyway.